src/wwwroot/app/directives/candidate-vacancy-info/candidate-vacancy-info.js

```diff
@@ -266,29 +425,28 @@ function CandidateVacancyInfoController($scope, // eslint-disable-line max-state
             return moment().format('DD-MM-YYYY');
```

Comment:
you use 'DD-MM-YYYY' format several times, so it has sence to define constant for this format;